static/app/views/profiling/landing/profilingTableCell.tsx

```diff
@@ -1,48 +1,77 @@
 import DateTime from 'sentry/components/dateTime';
 import Duration from 'sentry/components/duration';
+import Link from 'sentry/components/links/link';
 import {IconCheckmark, IconClose} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {Organization, Project} from 'sentry/types';
+import {defined} from 'sentry/utils';
 import {Container, NumberContainer} from 'sentry/utils/discover/styles';
+import {getShortEventId} from 'sentry/utils/events';
+
+import {generateFlamegraphRoute} from '../utils';
 
 import {TableColumn, TableDataRow} from './types';
 
-function ProfilingTableCell(
-  column: TableColumn,
-  dataRow: TableDataRow,
-  _rowIndex: number,
-  _columnIndex: number
-) {
-  const value = dataRow[column.key];
+function renderProfilingTableCell({
```

Comment:
@Zylphrex Do we need this to create a closure? I think we should stick to "component" approach as much as we can, because in the end, that's what we are rendering. The fact that we are rendering it as a function call here is unfortunate, but imo not something that the ProfilingTableCell should care about - that responsibility should fall to the component that is rendering the ProfilingTableCell.

So in this case, I think we should change this to 
```tsx
interface ProfilingTableCellProps {
  column: TableColumn;
  columnIndex: number;
  dataRow: TableDataRow;
  rowIndex: number;
}

function ProfilingTableCell(props: ProfilingTableCellProps) {
  const {projects} = useProjects();
  const organization = useOrganization();
  const value = props.dataRow[props.column.key];
  
  ....rest of body
}
```

and inside the profilingTable.tsx, create our render fn like so

```tsx
const renderProfilingTableCell: GridEditableProps<any, any>['grid']['renderBodyCell'] = (
  column,
  dataRow,
  rowIndex: number,
  columnIndex: number
) => {
  return (
    <ProfilingTableCell
      column={column}
      dataRow={dataRow}
      rowIndex={rowIndex}
      columnIndex={columnIndex}
    />
  );
};
```

And then pass it to the grid as we are doing already. This way we can keep our ProfilingTableCell unopinionated about how others components decide to render it and keep the specific render proxy fn closer to where it is used.